src/main/java/com/example/java/maven/cardGamePeterPan/Card.java

```diff
@@ -0,0 +1,57 @@
+package com.example.java.maven.cardGamePeterPan;
+
+public class Card {
+    private final Suit suit;
+    private final Rank rank;
+
+
+    public Card(Suit suit, Rank rank) {
+        this.suit = suit;
+        this.rank = rank;
+    }
+
+    public Suit getSuit() {
+        return suit;
+    }
+
+    public Rank getRank() {
+        return rank;
+    }
+
+    @Override
+    public String toString() {
+        return suit.symbol + rank.figure;
+    }
+
+    public enum Suit {
+
+        CLUB("♣"),
+        DIAMOND("♦"),
+        HEART("♥"),
+        SPADE("♠"),
+        WITCH("");
```

Comment:
I am not sure about this solution - that you have WITCH as `Suit` and `Rank` it is a little bit confusing.